static/webpack.config.js

```diff
@@ -21,7 +21,8 @@ const FixStyleOnlyEntriesPlugin = require("webpack-fix-style-only-entries");
 const config = {
   entry: {
     choropleth: [
-      __dirname + "/js/tools/choropleth/choropleth.ts",
+      __dirname + "/js/tools/choropleth/choropleth.tsx",
+      __dirname + "/js/tools/choropleth/choropleth_template.tsx",
```

Comment:
Only one end point should be necessary. Take a look at our other components for an example.

Reply:
Removed the template